raiden/tests/README.rst

```diff
@@ -1,6 +1,6 @@
 Running integration tests with different transport protocols
 ============================================================
 
-Raiden can be run with two different underlying transport protocols, UDP and Matrix. Therefore tests that depend on the transport layer (all of them are integration tests) come in two versions.
+Raiden can be run with Matrix underlying transport protocols. Therefore tests that depend on the transport layer (all of them are integration tests) come in two versions.
 
-The pytest option ``--transport=none|udp|matrix|all`` can be used to specify which versions of the test to run. By default, only the UDP versions of the tests are run, since the Matrix versions require the local installation of the `Synapse <https://matrix.org/docs/projects/server/synapse.html>`_ Matrix server.
+The pytest option ``--transport=none|matrix`` can be used to specify which versions of the test to run. By default, only the UDP versions of the tests are run, since the Matrix versions require the local installation of the `Synapse <https://matrix.org/docs/projects/server/synapse.html>`_ Matrix server.
```

Comment:
This can be removed entirely since synapse has been part of the dev requirements for some time now.